packages/main/src/AvatarGroup.js

```diff
@@ -245,8 +259,12 @@ class AvatarGroup extends UI5Element {
 		return this._itemsCount - this._hiddenItems;
 	}
 
+	get _useTotalAvatarsCount() {
```

Comment:
the convention is that boolean getters start with "should", so `_shouldUseTotalAvatarsCount`